test/integration/certs_test.go

```diff
@@ -0,0 +1,52 @@
+package integration_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/blocky/nitrite/internal"
+)
+
+func TestFetchingNitroCertProvider_Roots(t *testing.T) {
+	t.Run("happy path", func(t *testing.T) {
+		if testing.Short() {
+			t.SkipNow()
+		}
+
+		// given
+		cp := internal.NewFetchingNitroCertProvider()
+
+		// when
+		got, err := cp.Roots()
```

Comment:
Check that we can actually get new certs from the internet - requires network access.